### ISSUE TITLE
Final wiki walkthrough

### ISSUE THREAD

**Issue by sardred** [IT2901-24-2018/vapi#178]
Before leaving the repository behind, we should make sure the information in the wiki is correct and up to date. Here's a couple of things we should look into:

- [x] Mark pages that are no longer relevant as out of date or deprecated (read: pages about maps)
- [x] Remove mentions of React, ESLint and AirBnB that are no longer relevant
- [x] Revisit frontpage, make sure it gives most relevant information succinctly first
- [x] Read through everything for typos, or copy-paste text to docs or similar for spellcheck
- [x] Make sure all edge-cases we can think of are noted @Pontius1007  & @haavahe

**Comment by sardred** [comment]
Blocked by #179

**Comment by DjCreztian** [comment]
I figured the wiki pages about information for ex. Leaflet and map should be noted under a section possibilities for future developers (they would need the same projection etc.)
